src/plugins/data/common/storage/storage.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { transform, startsWith, keys } from 'lodash';
+import { parse, stringify } from '@osd/std';
+
+export enum StorageKeys {
+  WIDTH = 'widths',
+}
+
+type IStorageEngine = typeof window.localStorage;
+export class DataStorage {
```

Comment:
i do like this structure. it makes it easier to not use `opensearchDashboards.` and just pass the key